packages/ecmascript/src/Map.ts

```diff
@@ -14,7 +14,7 @@
  */
 
 export default class Map<K extends (string | number), V> {
-	private size = 0;
+	private size: number = 0;
```

Comment:
Shouldn't size be a public property?

Reply:
Actually not, as in the resulting code it would be public anyway.

Reply:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Map/size 

`size` should be public